packages/cli/src/ceramic-daemon.ts

```diff
@@ -968,6 +972,10 @@ export class CeramicDaemon {
       .json({ error: 'Method not supported: pin requests have moved to the admin API /admin/pins' })
   }
 
+  async _empty200(req: Request, res: Response): Promise<void> {
+    res.sendStatus(StatusCodes.OK)
```

Comment:
could we maybe still include a field in the response object with a warning message that this is deprecated and not doing anything?  People using tools are unlikely to see it, but at least if someone manually hits the pin endpoint they'll see something in the response object telling them what's up

Reply:
yeah that makes sense, adding now